src/script/calling/Call.ts

```diff
@@ -50,17 +49,23 @@ export class Call {
     this.conversationType = conversationType;
     this.initialType = callType;
     this.selfParticipant = selfParticipant;
-    this.participants = ko.observableArray();
+    this.participants = ko.observableArray([selfParticipant]);
     this.reason = ko.observable();
     this.startedAt = ko.observable();
   }
 
+  getSelfParticipant(): Participant {
+    return this.participants().find(
+      ({userId, deviceId}) => this.selfParticipant.userId === userId && this.selfParticipant.deviceId === deviceId,
+    );
+  }
+
   addParticipant(participant: Participant): void {
-    this.participants.unshift(participant);
+    this.participants.push(participant);
   }
 
-  getParticipant(userId: string): Participant {
-    return this.participants().find(participant => participant.userId === userId);
+  getParticipant(userId: UserId, deviceId: DeviceId): Participant {
```

Comment:
Let's call it `clientId`.

Reply:
I named it `deviceId` here, as the property on the `Participant` is also called `deviceId`

Reply:
So `DeviceId` should also be renamed.

Reply:
renamed everywhere